### DISCUSSION TITLE
v1.36.0

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#7150] [Changelog]
### New addons
- [Customizable menu bar](https://github.com/ScratchAddons/ScratchAddons/pull/6297) (`custom-menu-bar`) - replaces the hide tutorials button addon
- [Place backpack code at mouse](https://github.com/ScratchAddons/ScratchAddons/pull/7028) (`place-backpack-code-at-cursor`) - enabled by default
- [Rearrangeable custom block inputs](https://github.com/ScratchAddons/ScratchAddons/pull/7020) (`reorder-custom-inputs`)
- [Square block text inputs](https://github.com/ScratchAddons/ScratchAddons/pull/5331) (`editor-square-inputs`)

### Existing addons
- [Auto-hiding block palette: New "open by default" setting](https://github.com/ScratchAddons/ScratchAddons/pull/7076)
- [Confirmations: The comment setting now confirms when clicking reply](https://github.com/ScratchAddons/ScratchAddons/pull/7018)
- [Customizable link style: Separate bold setting and better settings layout](https://github.com/ScratchAddons/ScratchAddons/pull/6653)
- [Infinite scrolling: Support the explore and search pages](https://github.com/ScratchAddons/ScratchAddons/pull/6813)
-  [Enhanced full screen: Add "show toolbar when hovered" setting when hiding the toolbar in full screen](https://github.com/ScratchAddons/ScratchAddons/pull/6034)

### Settings page
- [Show IDs when addon names are hovered on development versions](https://github.com/ScratchAddons/ScratchAddons/pull/6786)

### Addons
-  [60FPS project player mode: Rename to "Higher project framerate mode" and improve description](https://github.com/ScratchAddons/ScratchAddons/pull/6848)
-  [Arrow key incrementation in editor: Use BigInt](https://github.com/ScratchAddons/ScratchAddons/pull/6794)
- [Block switching: Rename "arguments" in settings to "parameters"](https://github.com/ScratchAddons/ScratchAddons/pull/6979)
- [Customizable block text style: improve description](https://github.com/ScratchAddons/ScratchAddons/pull/6828)
- [Customizable link style: Inject styles sooner](https://github.com/ScratchAddons/ScratchAddons/pull/7132)
-  [Gamepad support: Remove warning on Linux Firefox 123+](https://github.com/ScratchAddons/ScratchAddons/pull/7090)
- [HD image uploads: Rename "stage" in settings to "canvas"](https://github.com/ScratchAddons/ScratchAddons/pull/6759)
-  [Preview project instructions and notes: Simplify implementation and improve reliability](https://github.com/ScratchAddons/ScratchAddons/pull/6893)
-  [Project video recorder: Better video encoding and add fallbacks](https://github.com/ScratchAddons/ScratchAddons/pull/7058)
- [Use `event.key` instead of `code` and `keyCode`](https://github.com/ScratchAddons/ScratchAddons/pull/6923)

### Popup and settings page
- [Add beta tag to deprecated/broken forum addons](https://github.com/ScratchAddons/ScratchAddons/pull/7183)
- [Adjust which addons are categorized as themes](https://github.com/ScratchAddons/ScratchAddons/pull/7050)
- [Popup: Default to the addons tab instead of messaging](https://github.com/ScratchAddons/ScratchAddons/pull/6781)
-  [Popup: Add reduced motion support to the permissions popup](https://github.com/ScratchAddons/ScratchAddons/pull/6725)
- [Settings: Improve selected category and toggle contrast](https://github.com/ScratchAddons/ScratchAddons/pull/6240)

### Other
- [Confirmations: Add code comments](https://github.com/ScratchAddons/ScratchAddons/pull/7017)
- [get-addon-settings.js: Add code comments](https://github.com/ScratchAddons/ScratchAddons/pull/6981)
- [Repository: Improve CONTRIBUTING.md](https://github.com/ScratchAddons/ScratchAddons/pull/6639)
- [Repository: Specify version for Prettier action](https://github.com/ScratchAddons/ScratchAddons/pull/6897)

### Addons
- [Add wildcards to /mystuff URL matches](https://github.com/ScratchAddons/ScratchAddons/pull/6962)
- [Better forum quoter: Escape square brackets](https://github.com/ScratchAddons/ScratchAddons/pull/6816)
- [Better forum quoter: Fix quoting the rolleyes emoji](https://github.com/ScratchAddons/ScratchAddons/pull/7126)
-  [Capitalized Account Settings: Fix on teacher accounts](https://github.com/ScratchAddons/ScratchAddons/pull/7149)
- [Colored context menus: Support high contrast mode](https://github.com/ScratchAddons/ScratchAddons/pull/6903)
- [Compact editor: Don't reduce the paint tools to one column there isn't enough space](https://github.com/ScratchAddons/ScratchAddons/pull/6894)
- [Ctrl+Enter to post: Fix when creating a new forum topic](https://github.com/ScratchAddons/ScratchAddons/pull/6927)
- [Customizable block colors: Fix labels in the custom block editor with some configurations](https://github.com/ScratchAddons/ScratchAddons/pull/7031)
- [Customizable block colors: Fix transparent inputs](https://github.com/ScratchAddons/ScratchAddons/pull/6902)
- [Customizable block shape: fix dropdown arrow position](https://github.com/ScratchAddons/ScratchAddons/pull/6905)
- [Customizable block shape: fix hacked empty inputs](https://github.com/ScratchAddons/ScratchAddons/pull/6906)
- [Customizable navigation bar: Hide username with CSS](https://github.com/ScratchAddons/ScratchAddons/pull/6889)
- [Editor dark mode and customizable colors: Fix dangerous context menu items](https://github.com/ScratchAddons/ScratchAddons/pull/7048)
- [Editor dark mode and customizable colors: Style WebGL modal correctly](https://github.com/ScratchAddons/ScratchAddons/pull/7038)
- [Editor find bar: Translate "flag"](https://github.com/ScratchAddons/ScratchAddons/pull/7056)
- [Emoji picker for comments: Fix layering issues with the report and delete buttons](https://github.com/ScratchAddons/ScratchAddons/pull/6890)
- [Faster project loading: Fix dropping code into sprite-pane](https://github.com/ScratchAddons/ScratchAddons/pull/7162)
- [Emoji picker for comments: Fix margins](https://github.com/ScratchAddons/ScratchAddons/pull/6811)
- [Insert blocks by name: Fix showing on the project page](https://github.com/ScratchAddons/ScratchAddons/pull/7032)
- [Onion skinning and costume editor snapping: Close modals when clicking outside of them](https://github.com/ScratchAddons/ScratchAddons/pull/6801)
-  [Pause button: Fix keyboard shortcut on MacOS](https://github.com/ScratchAddons/ScratchAddons/pull/7131)
- [Save blocks as image: Properly support the high contrast mode](https://github.com/ScratchAddons/ScratchAddons/pull/6907)
- [Scratch 2.0 → 3.0: Resize profile rows when dynamically enabling or disabling](https://github.com/ScratchAddons/ScratchAddons/pull/6924)
-  [Scratch 2.0 → 3.0: Style comment warnings correctly](https://github.com/ScratchAddons/ScratchAddons/pull/6873)
- [Search bar on My Stuff: Don't show when disabled](https://github.com/ScratchAddons/ScratchAddons/pull/6996)
- [Sprite folders: Place duplicated sprites in the correct position](https://github.com/ScratchAddons/ScratchAddons/pull/6921)
- [Sticky footer: Fix layering issues with "old studio layout"](https://github.com/ScratchAddons/ScratchAddons/pull/6969)

### Other
- [Fix addon userscripts using wrong language for localized strings](https://github.com/ScratchAddons/ScratchAddons/pull/6802)
- [Readme: Fix Chrome documentation link](https://github.com/ScratchAddons/ScratchAddons/pull/6947)
- [Readme: Fix Markdown note block](https://github.com/ScratchAddons/ScratchAddons/pull/6878)

I don't have access to the changelog category, could someone else move it there?

**Comment by Hans5958** [comment]
Saw that you tried to do a changelog different that what Apple does. Might as well do a suggestion.

I think you should try doing a two-level separation just like what I did on [`changelog-detailed`](https://github.com/ScratchAddons/ScratchAddons/blob/changelog-detailed/CHANGELOG.md). It is based on [Keep a Changelog](https://keepachangelog.com), though you shouldn't follow it strictly and do something creative to make it better.

**Reply by Samq64** [reply]
That does make things look a little cleaner.
